### ISSUE TITLE
How to use transformers library?

### ISSUE THREAD

**Issue by edchengg** [learnables/learn2learn#187]
Hi, I would like to use a pre-trained language model (BERT) from the transformers library to do NLP tasks.
Can I use models such as BertForSequenceClassification directly?

From what I see in the https://github.com/learnables/learn2learn/blob/master/examples/text/news_topic_classification.py, learn2learn uses fairseq roberta to extract features from text and feed it into Net.
It sets 'roberta' to eval() model. But I would like to finetune the "roberta" during training. Can we do that?

I am not sure how learn2learn calculates meta-gradient. Can you also point me to the code?

Thank you!

**Comment by seba-1511** [comment]
Several people have tried using transformers with MAML, but few were successful. @dadelani, do you have any advice?

**Comment by dadelani** [comment]
I have not successfully implemented it using "BertForSequenceClassification". What seems to work for me is using frozen BERT + LSTM + linear classifier, please see 
https://colab.research.google.com/drive/1qjSXnNSynsZQICI6j1QuLP-E8m5xN2yx?usp=sharing
